codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceGenerator.java

```diff
@@ -152,9 +155,7 @@ func NewClient(clientConfig $L) (*$T, error) {
           operation,
           OperationShape.class
         );
-        final var inputShape = model.expectShape(
-          operationShape.getInputShape()
-        );
+        var inputShape = model.expectShape(operationShape.getInputShape());
```

Comment:
I honestly did not know that Java had `var` keyword.
Apparently, I have only ever written for Java 1.8, and Java 10 introduced `var`.
Wild.
(I hate it. Strong typing for the win. But I am not blocking/suggesting changes for it.)

Reply:
I discovered `var` last year when I started this project. I was in the camp Strong typing but then realized more and more languages are dynamically typed (at least in syntactical sense). Plus this is not a prod code. (I still prefer production have the types just for readability/debugging)